cmd/neofs-node/attributes.go

```diff
@@ -11,4 +15,52 @@ func parseAttributes(c *cfg) {
 	}
 
 	fatalOnErr(attributes.ReadNodeAttributes(&c.cfgNodeInfo.localInfo, nodeconfig.Attributes(c.appCfg)))
+
+	// expand UN/LOCODE attribute if any found; keep user's attributes
+	// if any conflicts appear
+
+	locAttr := c.cfgNodeInfo.localInfo.LOCODE()
+	if locAttr == "" {
+		return
+	}
+
+	key, record, err := getRecord(locAttr)
+	if err != nil {
+		fatalOnErr(fmt.Errorf("could not get locode record from DB: %w", err))
+	}
+
+	n := &c.cfgNodeInfo.localInfo
+
+	setLocodeAttr(n, "CountryCode", key.CountryCode().String())
+	setLocodeAttr(n, "Country", record.Country)
+	setLocodeAttr(n, "Location", record.Location)
+	setLocodeAttr(n, "Continent", record.Cont.String())
+	if subDivCode := record.SubDivCode; subDivCode != "" {
+		setLocodeAttr(n, "SubDivCode", subDivCode)
+	}
+	if subDivName := record.SubDivName; subDivName != "" {
+		setLocodeAttr(n, "SubDiv", subDivName)
+	}
+}
+
+func getRecord(lc string) (*locodedb.Key, locodedb.Record, error) {
+	country, location := lc[:2], lc[2:]
+	if lc[2] == ' ' {
+		location = lc[3:]
```

Comment:
Doing these dances just for `locodedb.NewKey()` is no fun.

Reply:
copy-pasted from the approved PR

Reply:
You can add some `locodedb.CodeToKey(string)`. It's always to possible to do something better than the nearest PR. And we should be doing that.

Reply:
> You can add some locodedb.CodeToKey(string)

you mean locode repo?

> It's always to possible to do something better than the nearest PR.

i agree but do not know what we can do here. removed that check and used country code constants instead

Reply:
> you mean locode repo?

Sure.

Reply:
5 char len string to some type to 2 char len string (same but shorter) then?

Reply:
OK this way too, you get the data first which ensures that locode is correct and its first two bytes can be used this way safely.